README.md

```diff
@@ -76,6 +78,7 @@ class Example extends React.Component {
               {pattern: /#(\w+)/,                 style: styles.hashTag},
             ]
           }
+          childrenProps={{ allowFontScaling: false }}
```

Comment:
Should follow the pattern in the rest of the code & readme by removing the outer spaces